### PR TITLE
indexing: nit-picks

### PR DESCRIPTION
**Pull request by coszio**
A few nit-pick suggestions on `indexing.md`

### CODE REVIEW

File: qdrant/v1.1.x/indexing.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Indexing
 weight: 90
 ---
 
-A key feature of Qdrant is the effective combination of vector and traditional indices. It is essential to have this because for vector search to work effectively with filters, having vector index only is not enough.
+A key feature of Qdrant is the effective combination of vector and traditional indexes. It is essential to have this because for vector search to work effectively with filters, having vector index only is not enough. In simpler terms, a vector index improves vector search, and payload indexes improve filtering.
```
Comment (generall) [inline qdrant/v1.1.x/indexing.md:6]: maybe improves -> speeds up?

**Review by generall — APPROVED** [review]

**Review by agourlay — APPROVED** [review]
